test/test_images/recordevents/eventstore.go

```diff
@@ -73,21 +73,36 @@ func (es *eventStore) checkAppendBlock() {
 }
 
 // Store the specified event.
-func (es *eventStore) StoreEvent(event cloudevents.Event, httpHeaders map[string][]string) {
+func (es *eventStore) StoreEvent(event *cloudevents.Event, evErr error, httpHeaders map[string][]string) {
```

Comment:
are you sure we need all http headers? Can't we have just the non ce headers (aka the ones not prefixed with `ce-`)?

Reply:
yeah, it's probably fine to just strip those out.